.circleci/config.yml

```diff
@@ -17,12 +39,25 @@ jobs:
       - run: scripts/qa/unused.sh
       - run: scripts/qa/vendor.sh
       - run: go vet ./...
-      - run: scripts/build.sh
-      - run: scripts/build_tools.sh
-      - run: scripts/build_docker.sh
+
+  qa-post-build:
+    working_directory: /home/circleci/.go_workspace/src/github.com/grafana/metrictank
+    machine: true
+    steps:
+      - checkout
+      - attach_workspace:
+          at: .
       - run: scripts/qa/docs.sh
-      - run: go test -v -race $(go list ./... | grep -v github.com/grafana/metrictank/chaos)
+      - run: docker load -i build_docker/metrictank.tar
       - run: scripts/qa/end2end.sh
+
+  deploy:
+    docker:
+      - image: circleci/ruby:2.3
+    steps:
+      - checkout
+      - attach_workspace:
+          at: build
```

Comment:
I think this should be `at: .`  otherwise you will end up with `build/builld/metrictank`